config/owasp/suppressions.xml

```diff
@@ -17,4 +17,14 @@
   <suppress until = "2023-09-01">
```

Comment:
probably remove once which are not relevant anymore ?

Reply:
Have removed the old suppression